### ISSUE TITLE
Ensure pkg_ref_cache.covr_coverage.pkg_source points to appropriate path to the package

### ISSUE THREAD

**Issue by ndelcamp** [pharmaR/riskmetric#112]
The call to `covr::package_coverage()` does not include the `path` argument, which has default value `"."`. Since users will likely not have the source code in their working directory, I think line 14 of R/pkg_ref_cache_covr_coverage.R should point to the `pkg_ref` path: 
```R
covr::package_coverage(path = x$path, type = "none", code = deparse(expr))
```
Example to reproduce error:
```R
library(riskmetric)
source_code_path <- "source-code/acepack_1.4.1/acepack/"
ref <- pkg_ref(source_code_path)
ref$covr_coverage
# Error: ‘/home/ndelcamp/risk-assessments’ does not contain a package!

# Results are expected to match:
expr <- bquote(tools::testInstalledPackage("acepack", types = 'tests'))
covr::package_coverage(path = source_code_path, type = "none", code = deparse(expr))
# acepack Coverage: 33.57%
# R/zzz.R: 0.00%
# src/rlsmo.f: 0.00%
# R/acepack.R: 17.88%
# src/avas.f: 34.32%
# src/ace.f: 59.89%
```
I did test the change on my repo, which resulted in the expected result.

**Comment by dgkf** [comment]
Fantastic - great catch. That's my oversight. I'll get this fixed as soon as possible.

**Comment by dgkf** [comment]
This is now fixed in master as of #113
